src/hooks/useHtmlPaste/index.ts

```diff
@@ -12,6 +12,7 @@ const insertAtCaret = (target: HTMLElement, text: string) => {
     if (selection?.rangeCount) {
         const range = selection.getRangeAt(0);
         range.deleteContents();
+        text = text.replace(/^[\r\n]+|[\r\n]+$/g, '');  // Remove trailing Carriage Return and Line feed
```

Comment:
Define this regex in the [CONST](https://github.com/Expensify/App/blob/8375abea3a377e7416404addde255b6ac8530a9b/src/CONST.ts#L1894) file

Reply:
Fixed